hw/ip/rv_plic/doc/checklist.md

```diff
@@ -0,0 +1,232 @@
+[SIM_CSR_MEM_TEST_SUITE_PASSING]:     {{<relref "/doc/project/checklist.md#sim-csr-mem-test-suite-passing" >}}
+[FPV_MAIN_ASSERTIONS_PROVEN]:         {{<relref "/doc/project/checklist.md#fpv-main-assertions-proven" >}}
+[SIM_ALT_TOOL_SETUP]:                 {{<relref "/doc/project/checklist.md#sim-alt-tool-setup" >}}
+[SIM_SANITY_REGRESSION_SETUP]:        {{<relref "/doc/project/checklist.md#sim-sanity-regression-setup" >}}
+[SIM_NIGHTLY_REGRESSION_SETUP]:       {{<relref "/doc/project/checklist.md#sim-nightly-regression-setup" >}}
+[FPV_REGRESSION_SETUP]:               {{<relref "/doc/project/checklist.md#fpv-regression-setup" >}}
+[SIM_COVERAGE_MODEL_ADDED]:           {{<relref "/doc/project/checklist.md#sim-coverage-model-added" >}}
+[PRE_VERIFIED_SUB_MODULES_V1]:        {{<relref "/doc/project/checklist.md#pre-verified-sub-modules-v1" >}}
+[DESIGN_SPEC_REVIEWED]:               {{<relref "/doc/project/checklist.md#design-spec-reviewed" >}}
+[DV_PLAN_TESTPLAN_REVIEWED]:          {{<relref "/doc/project/checklist.md#dv-plan-testplan-reviewed" >}}
+[STD_TEST_CATEGORIES_PLANNED]:        {{<relref "/doc/project/checklist.md#std-test-categories-planned" >}}
+[V2_CHECKLIST_SCOPED]:                {{<relref "/doc/project/checklist.md#v2-checklist-scoped" >}}
+
+### Checklists for milestone V2
+
+ Type         | Item                                    | Resolution  | Note/Collaterals
+--------------|-----------------------------------------|-------------|------------------
+Documentation | [DESIGN_DELTAS_CAPTURED_V2][]           | N/A         |
+Documentation | [DV_PLAN_COMPLETED][]                   | Done        |
+Testbench     | [ALL_INTERFACES_EXERCISED][]            | Done        |
+Testbench     | [ALL_ASSERTION_CHECKS_ADDED][]          | Done        |
+Testbench     | [SIM_TB_ENV_COMPLETED][]                | N/A         |
+Tests         | [SIM_ALL_TESTS_PASSING][]               | N/A         |
+Tests         | [FPV_ALL_ASSERTIONS_WRITTEN][]          | Done        |
+Tests         | [FPV_ALL_ASSUMPTIONS_REVIEWED][]        | Done        |
+Tests         | [SIM_FW_SIMULATED][]                    | N/A         |
+Regression    | [SIM_NIGHTLY_REGRESSION_V2][]           | N/A         |
+Coverage      | [SIM_CODE_COVERAGE_V2][]                | N/A         |
+Coverage      | [SIM_FUNCTIONAL_COVERAGE_V2][]          | N/A         |
+Coverage      | [FPV_CODE_COVERAGE_V2][]                | Done        |
+Coverage      | [FPV_COI_COVERAGE_V2][]                 | Done        |
+Issues        | [NO_HIGH_PRIORITY_ISSUES_PENDING][]     | Done        |
+Issues        | [ALL_LOW_PRIORITY_ISSUES_ROOT_CAUSED][] | Done        |
+Integration   | [PRE_VERIFIED_SUB_MODULES_V2][]         | N/A         |
+Review        | [V3_CHECKLIST_SCOPED][]                 | Done        |
+Review        | Reviewer(s)                             | @eunchan    |
+Review        | Signoff date                            | 2020-02-03  |
```

Comment:
Since - who knows - we might end up saying V2 is sufficient for some modules
for ES completion, I would like to see one other DVer signoff on V2. @weicaiyang 
or @sriyerg ? Just to review the test plan and test lists.